src/main/java/org/broadinstitute/hellbender/tools/exome/gcbias/CorrectGCBias.java

```diff
@@ -17,24 +18,36 @@
 /**
  * Correct coverage for sample-specific GC bias effects as described in {@link GCCorrector}.
  *
+ * <p>
  * Inputs are
  * 1. read counts file (format described in {@link ReadCountCollectionUtils}).  Since this tool corrects
  *    for multiplicative GC bias effects, these counts should not be in log space i.e. they should represent
  *    raw coverage or proportional coverage upstream of tangent normalization.
  * 2. targets file containing GC content annotation as produced by {@link AnnotateTargets}.  Every target in the input
  *    read counts must be present in the targets file but the reverse need not be true.
  *
- * Output is a read counts file with the same targets (rows) and samples (columns) as the input, corrected for GC bias.
+ * The resulting read counts file has the same targets (rows) and samples (columns) as the input and GC-bias-corrected coverage.
  * Coverage is represented by doubles in {@link ReadCountCollection}.
+ * </p>
+ *
+ * <h3>Example</h3>
+ *
+ * <pre>
+ * java -Xmx4g -jar $gatk_jar CorrectGCBias \
+ *   --input coverage.tsv \
+ *   --targets targets.annotated.tsv \
+ *   --output coverage.gc_corrected.tsv
+ * </pre>
  *
  * @author David Benjamin &lt;davidben@broadinstitute.org&gt;
  */
 @CommandLineProgramProperties(
-        oneLineSummary = "Correct for per-sample GC bias effects",
```

Comment:
There are arguments for both "per-sample" and "per-target".  Roughly speaking, GC bias is a multiplicative bias that affects coverage at a target as a function of the GC content of that target, but the particular function differs from sample to sample (as is indicated by the `summary` line below).  I think "Correct for sample-specific GC bias" is fine for the `oneLineSummary`.